### DISCUSSION TITLE
Change WebUI color scheme

### DISCUSSION THREAD

**Discussion by Revell1on** [transmission/transmission#5599] [Q&A] [answered]
Hello, I am using transmission installed on my router through asuswrt-merlin and entware, recently i updated transmission 3 to 4 and received a dark color scheme. I would like to switch back to the old interface of 3d version if possible, it was much more accurate and prettier. How can I do that?
If this is impossible, i'd like to ask if there is an option to get myself at least a light color scheme for the new interface

**Comment by Pentaphon** [comment]
I think that UI is long gone. You can try 3rd party WebUIs from https://transmissionbt.com/addons

**Reply by Revell1on** [reply]
This answers only part of my questions. The other part was about the way of forcing light scheme

**Reply by Pentaphon** [reply, marked as answer]
Give https://github.com/openscopeproject/TrguiNG a try. It's the nicest looking WebUI, I think, and it lets you pick dark or light theme.
